code/controllers/subsystems/jobs.dm

```diff
@@ -505,7 +498,7 @@ SUBSYSTEM_DEF(jobs)
 	// If they're head, give them the account info for their department
 	if(H.mind && job.head_position)
 		var/remembered_info = ""
-		var/datum/money_account/department_account = department_accounts[job.department]
+		var/datum/money_account/department_account = department_accounts[job.department_refs[1]]
```

Comment:
May be better to have some `primary_department` var on the job, automatically assigned the first entry in their `department_refs` on instantiation if not set manually.

Reply:
Yup I see that. The code that checks for that should go in SSdeparment init right?

Reply:
I'd put it in job/Initialize() or job/New() personally, whichever currently holds the logic.